src/main/resources/templates/fragments/review/sectionHeader.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <title>Section Header</title>
+    </head>
+    <body>
+        <h2 th:fragment="sectionHeader" class="govuk-heading-m"><th:block th:text="${description}"></th:block>
```

Comment:
Are we going to use this fragment on other pages? If not, is it worth renaming to be review specific?

Reply:
I've added a `review` subdirectory to `fragments` rather than pollute the name of each fragment file. If there's a preference for filename prefixes or suffixes I can change this?